app/pages/HomePage/HomePage.tsx

```diff
@@ -27,6 +32,79 @@ export interface HeroData {
   buttons: ButtonType[];
 }
 
+interface HomePageData {
+  heroSection: HeroData;
+  categoriesSection: FeaturedCategoriesData;
+  opportunitySection: OppEventCardData;
+  eventSection: OppEventCardData;
+  /* Fix naming in Sanity schema */
+  twoColumnContentSections: {
+    twoColumnContentSections: TwoColumnContent;
+  }[];
+}
+
+export const HomePage = () => {
+  const [homePageData, setHomePageData] = useState<HomePageData | null>(null);
+
+  useEffect(() => {
+    const fetchHomePageData = async () => {
+      const query = `*[_type == "homePage" && name == "Home Page"][0]{
+        'heroSection': *[_type == "hero" && name == "Home Hero"][0],
+        categoriesSection {...,'featuredCategoriesSection': featuredCategoriesSection[]->},
+        opportunitySection {...,'opportunityCards': opportunityCards[]->},
+        eventSection {...,'eventCards': eventCards[]->},
+        'twoColumnContentSections': *[_type == 'contentPageType' && name == 'Home']{
+      twoColumnContentSections[0]->
+    }
+      }`;
+
+      const result: HomePageData = await client.fetch(query);
```

Comment:
Are we punting on error handling for the time being? That's ok, I'm just curious because if Sanity sent something back unexpected I have a sense this would brick the page.

Reply:
Added in error handling!